crates/common/consensus/lean/src/block.rs

```diff
@@ -1,22 +1,36 @@
-use alloy_primitives::B256;
+use ethereum_hashing::hash;
 use ream_pqc::PQSignature;
 use serde::{Deserialize, Serialize};
 use ssz_derive::{Decode, Encode};
-use tree_hash_derive::TreeHash;
+use ssz_types::{
+    VariableList,
+    typenum::{
+        U16777216, // 2**24
+    },
+};
 
-#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash)]
+use crate::{Hash, vote::Vote};
+
+// TODO: Add back #[derive(TreeHash)]
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode)]
 pub struct SignedBlock {
     pub message: Block,
     pub signature: PQSignature,
 }
 
-#[derive(
-    Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash, Default,
-)]
+// TODO: Add back #[derive(TreeHash)]
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, Default)]
 pub struct Block {
```

Comment:
why was u64 changed to usize? also we should use B256 instead of the type alias

Reply:
There's quite a bit of vector manipulation which I think requires usize.

But I updated the code now to u64 and do `as usize` only when working with vectors.